src/oscar/templates/oscar/dashboard/catalogue/product_update.html

```diff
@@ -81,6 +81,7 @@ <h3>{% trans "Sections" %}</h3>
                                 {% if not parent %}
                                     <li><a href="#product_recommended" data-toggle="tab">{% trans 'Upselling' %}</a></li>
                                 {% endif %}
+                                <li><a href="#seo" data-toggle="tab">{% trans 'Search Engine Optimisation' %}</a></li>
```

Comment:
This is getting a bit nitpicky now, but can we use "Search engine optimisation" (lower case e,o) everywhere just for consistency with all the other tab titles?